src/indigo/less/forms.less

```diff
@@ -1,204 +1,53 @@
-
-label {
-    font-weight: 400;
-}
-
-.form-horizontal {
-    @media (min-width: @screen-sm-min) {
-        .control-label {
-            text-align: left;
-            font-weight: 400;
-            font-size: 120%;
-            color: #282d38;
-        }
-    }
+.form-control {
+  box-shadow: none;
 }
 
-textarea.form-control {
-    padding: 10px 15.6px;
+.input-group-addon {
+  background: darken(@input-bg, 5%);
 }
 
+.checkbox,
```

Comment:
Toto vzdy radsej vidim default. Velakrat to prestylovanie nedopoadne dobre. Ale uvidime. :)

Reply:
Jasné no. Je to v návrhu tak jsme se nějak snažil. Ale taky mám zkušenosti že upravovat takové prvky dopadá špatně, že to často není pak konzistentní.